Gemfile

```diff
@@ -2,10 +2,7 @@ source 'https://rubygems.org/'
 
 gemspec
 
-# https://github.com/socketry/async-io/blob/v1.23.1/async-io.gemspec#L21
-if Gem::Version.create(RUBY_VERSION) >= Gem::Version.create('2.3.0')
-  gem 'async-http', '~> 0.42'
-end
+gem 'async-http', '~> 0.42'
```

Comment:
this gem should be moved into fluentd.gemspec

Reply:
I think fluentd should work with minimal dependencies. So make async-http optional seems good for me. If we can replace cool.io with async gems, async-http should be default dependency.

Reply:
If we can replace cool.io with async gems, async-http should be default dependency.

Make sense 👍